### ISSUE TITLE
Gnaw(ghoul stun) and ghoul's GCD

### ISSUE THREAD

**Issue by Johanes022** [dalaranwow/dalaran-wow#5135]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Gnaw is on global CD wich wich does not allow you to use it when you want but instead you have to time it right between pets GCD aka spam it before ghoul uses his abilities.

**Expected behaviour**: Gnaw should not trigger ghoul’s GCD. This bug makes it hard to use Corpse Explosion, because if u cast CE while pet is on GCD, it won’t go off and u will waste 40 runic power.

**Steps to reproduce the problem**:

1. Attack your target with a pet. Use claw and observe that Gnaw triggers GCD
2. 
3. 

**Include proofs for this behaviour**

https://www.youtube.com/watch?v=mOPpsUcgRnY
http://www.warcraftmovies.com/movieview.php?id=160059  (same video as previous link)
time 3:13,  18:05   watch pet's gcd. the dk is able to cast CE right after using gnaw. 

https://youtu.be/QXR5sMKcn08?t=1565

Related issue: https://github.com/dalaranwow/dalaran-wow/issues/3548

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Johanes022,
Issue confirmed on PTR.
![image](https://user-images.githubusercontent.com/6797611/33515251-2b94857c-d769-11e7-9c93-79ef4a40bae6.png)

**Comment by Rushor** [comment]
@TeridanDW - https://github.com/TrinityCore/TrinityCore/issues/6056 describes the same issue no?

**Comment by TeridanDW** [comment]
No, I don't think so. @robinsch fixed the 40 energy issue and autocast problems in diffrent report, to my knowledge. The issue here is that if the pet has claw set on autocast player has hard time casting pet's spell Gnaw as he gets error caused by pet's global cd.
I'm pretty sure the gnaw should be castable even if the pet is on gcd, but will test it tomorrow on  retail, i have dk there.

**Comment by TeridanDW** [comment]
Here is a short video from retail:
https://www.youtube.com/watch?v=kneUTm4W3Hs&feature=youtu.be
From what I see, I was right in the comment above.

**Comment by Edzero** [comment]
Still broken on the ghoul. Nice to see things never change, eh?

Also corpse explosion is being affected by enemy resistances, even if you have spell penetration.

I assume it's because corpse explosion is being counted as damage from the pet rather than the player. Even so, pets should be inheriting spell penetration from their owner. Seems that is also not working.

#3088
